### PR TITLE
Feature/desktop mode docu trng 1438

### PR DESCRIPTION
**Pull request by tomwim**
### Description
Documentation for Desktop Mode

### CODE REVIEW

File: .Documentation/articles/pro/03-desktop-mode.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+[![Interaction Rig](../images/pro/03-rig-loader.png "Interaction rig settings.")](../images/pro/03-rig-loader.png)
+
+Moving the _Novice Desktop Mode_ rig to the top will ensure that the desktop mode (camera and interactions) will be loaded. Choosing any XR rig (_XR Rig_ is recommended) will load a VR configured camera rig and controllers as soon as the application starts.
+
+_Note:_ For testing in Unity you can switch between settings, but before you build your final application make sure you have the desired starting configuration set in both objects.
+
+## How to use the _novice_ Desktop Mode
+
+The novice Desktop mode is targeted on 'conservative' users, not used to playing computer games or to any advanced interaction with keyboard and mouse. We intentionally reduced the freedom of control to reduce complexity and increased the usability by simplifying the interaction. We will soon release the _advanced_ desktop mode providing more control.
```
Comment (Gusinuhe) [inline .Documentation/articles/pro/03-desktop-mode.md:25]: `We will soon release the _advanced_ desktop mode providing more control.`

Either we update this as soon as the advance mode is released or we just never publish it, this is not a newsletter.

**Review by Gusinuhe — CHANGES_REQUESTED** [review]
The article is not accessible in the articles section:

Screenshot 2021-02-22 at 21 42 43

File: .Documentation/articles/pro/03-desktop-mode.md (hunk 1 above)
Comment (tomwim) [reply .Documentation/articles/pro/03-desktop-mode.md]: You are right. Removed that line but kept the section on the bottom of the page where we say that an advanced mode will come. is that fine with you?

Comment (Gusinuhe) [reply .Documentation/articles/pro/03-desktop-mode.md]: That works for me 👍

**Review by Gusinuhe — DISMISSED** [review]

File: .Documentation/articles/pro/03-desktop-mode.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Desktop Mode
+
+The desktop mode is part of the Innoactive Creator Pro and allows you to run your training applications without the need of a VR headset. You can perform the training using keyboard and mouse on a regular desktop PC or laptop. The training course created in Innoactive Creator does not need to be adapted to run in desktop mode. Just build your training application for both VR and non-VR and you can switch even during runtime between a training experience in VR and a training experience using keyboard and mouse.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/pro/03-desktop-mode.md:3]: never used my keyboard for desktop mode.

File: .Documentation/articles/pro/03-desktop-mode.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Switching between modes
+
+You can switch between VR-mode and desktop mode through the trainer menu on the bottom of your screen. Switching the mode will reload your current scene and switching is not possible while a training course is running.
+
+[![Switching modes](../images/pro/03-trainer-menu.png "Switching between VR and Desktop Mode.")](../images/pro/03-trainer-menu.png)
+
+## Advanced Mode
+
+An advanced desktop mode with more features and more possibilities to interact with your scene will come soon.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/pro/03-desktop-mode.md:58]: I don't like that we are promising stuff "very soon". Cant we just say, it is planned or not mention it?

**Review by SimonTheSourcerer — APPROVED** [review]
We don't mention what is not working with the DM right now... should we do that?

File: .Documentation/articles/pro/03-desktop-mode.md (hunk 3 above)
Comment (tomwim) [reply .Documentation/articles/pro/03-desktop-mode.md:58]: it is not very soon, just soon :P but ill change it

Comment (Gusinuhe) [reply .Documentation/articles/pro/03-desktop-mode.md]: Maybe change it to "in a future release", that way makes sense if you read this in 2035 where you can just switch to a higher docu version.

**Review by SimonTheSourcerer — DISMISSED** [review]

**Review by vued — COMMENTED** [review]
Can/should we add a "We have some prototyes ready for user testing. In case you are interested in trying them out and would like to give feedback, please contaxt XYZ." to the advanced desktop mode section? @tomwim

**Comment by tomwim** [comment]
> Can/should we add a "We have some prototyes ready for user testing. In case you are interested in trying them out and would like to give feedback, please contaxt XYZ." to the advanced desktop mode section? @tomwim

I think that is something for the newsletter not the documentation

**Review by SimonTheSourcerer — DISMISSED** [review]

**Review by vued — APPROVED** [review]

**Pull request merged**